.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "Plain JF Dotnet 8",
+    "image":"mcr.microsoft.com/devcontainers/dotnet:8.0-jammy",
+        //required to get skia to run
+	"postCreateCommand": "sudo apt-get update && sudo apt-get install -y libfontconfig1",
+    "postStartCommand": "dotnet restore; dotnet workload update; dotnet dev-certs https --trust",
+    "containerEnv": {
+        "JELLYFIN_NETWORKMANAGER_BINDALL": "true"
+    },
+    "features": {
```

Comment:
I'd add the same features as in [Vue](https://github.com/jellyfin/jellyfin-vue/blob/master/.devcontainer/devcontainer.json#L11C60-L11C60)

- Having a setup ready for Docker is always a good idea
- gh cli is useful for checking out external's contributor's PRs, firing actions, etc etc

Reply:
i can add docker-in-docker as its a great i dea but i dont think gh-cli is strictly nessary as i am currently writing this from vscode without it.